src/PolygonEcosystemToken.sol

```diff
@@ -15,7 +15,7 @@ contract PolygonEcosystemToken is ERC20Permit, AccessControlEnumerable, IPolygon
     bytes32 public constant CAP_MANAGER_ROLE = keccak256("CAP_MANAGER_ROLE");
     bytes32 public constant PERMIT2_REVOKER_ROLE = keccak256("PERMIT2_REVOKER_ROLE");
     address public constant PERMIT2 = 0x000000000022D473030F116dDEE9F6B43aC78BA3;
-    uint256 public mintPerSecondCap = 10e18; // 10 POL tokens per second
+    uint256 public mintPerSecondCap = 13.37e18;
```

Comment:
As discussed, I would suggest to add a few more tests to check which are the new emission boundaries or limits today. 

I have just made some fuzzing around this. As a conclusion, it seems that in case of frequent minting the system will reach the limit in less than 12 years (from 23 years) but there might be edge situations where we can mint a bigger supply just waiting a bunch of years without minting any token as max emission depends on last minting timestamp.

Summarizing, there were some important behavioral changes on the emission system and its limits by this small change, so I would suggest we run some few more simulations as well as fuzzing cases next time if possible.  Thanks!